13-decay-tree-fitter.md

```diff
@@ -42,8 +42,10 @@ dtt.Dstar.ConsD.daughtersToConstrain = ['D0']
 ```
 Note that you can constrain more than one intermediate state at once if that fits your decay. 
 
+When using the DecayTreeFitter in a DecayTreeTuple, all the variables created by the other TupleTools are not affected by the change, but some new variables are created, one set per DecayTreeFitter instance. Depending on whether the `Verbose` option is specified, the new variables are created for the head particle only or for the head particle and its daughters too. If the daughters are not stable particles and decay further, the daughters of the daughters have no new variables associated to them. In some cases it might be useful to make this information available too. For more information on that, see [this](http://svn.cern.ch/guest/lhcb/Erasmus/trunk/Phys/Bd2D0Kst/DaVinci/TupleBuilder.py) example of how to use the DecayTreeFitter via LoKi functors.
```

Comment:
if we keep the link to `TupleBuilder.py` I have two requests: can we make it a link that will work for ever (include a specific revision number maybe)? The fear is that it will stop containing an example and we still link to it.

The other thing is to give it a better name than [this]. Maybe just move the link to be the words "example of how to use DecayTreeFitter via LoKi functors"